.changeset/block-scripts-jsx-transform.md

```diff
@@ -0,0 +1,6 @@
+---
+"block-scripts": patch
+"block-template": patch
```

Comment:
What change have we made in block-template?

Reply:
we updated the version number of block scripts used there

Reply:
Oh IC. This change makes sense then!